Components/Toast/toast.css

```diff
@@ -0,0 +1,37 @@
+:root{
+    --BG:#262626;
```

Comment:
Suggestion: You can keep better names for the variables like eg: BG-LIGHT

Reply:
Actually, I forgot to change the colors to primary and secondary!